test/system/255-auto-update.bats

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests for automatically update images for containerized services
+#
+
+load helpers
+
+UNIT_DIR="/usr/lib/systemd/system"
+SNAME_FILE=$BATS_TMPDIR/services
+
+function setup() {
+    skip_if_remote "systemd tests are meaningless over remote"
+    skip_if_rootless
+
+    basic_setup
+}
+
+function teardown() {
+    while read line; do
+        if [[ "$line" =~ "podman-auto-update" ]]; then
+            echo "Stop timer: $line.timer"
+            systemctl stop $line.timer
+            systemctl disable $line.timer
+        else
+            systemctl stop $line
+        fi
+        rm -f $UNIT_DIR/$line.{service,timer}
+    done < $SNAME_FILE
+
+    rm -f $SNAME_FILE
+    basic_teardown
+}
+
+function generate_service() {
+    fake_img=$1
+    target_img=$2
```

Comment:
Almost-useless function parameter: it is (almost) always `quay.io/libpod/alpine:latest`. There is only one exception, `alpine_nginx`, and there is no explanation of why that one is different.

If they can always all be the same, please remove this parameter.

If there is an important reason for using `alpine_nginx` in one instance, then (1) add a comment explaining the one-time use of nginx, and (2) please just make the function accept the basename (`alpine`, `alpine_nginx`) and have the function add the registry/tag:
```
    target_img=quay.io/libpod/${target_img_basename}:latest
```